server/src/utilities/jwtUtils.ts

```diff
@@ -0,0 +1,23 @@
+import jwt, { Secret } from 'jsonwebtoken';
+
+const { PRIVATE_KEY } = process.env;
+
+const jwtSign = (payload: { id: number; isAdmin: boolean }) => new Promise((resolve, reject) => {
```

Comment:
why there is a `;` here ??
```suggestion
const jwtSign = (payload: { id: number, isAdmin: boolean }) => new Promise((resolve, reject) => {
```